content/08_fractals.html

```diff
@@ -452,8 +452,8 @@ <h3 id="exercise-84">Exercise 8.4</h3>
   <h3 id="exercise-85">Exercise 8.5</h3>
   <p>Use recursion to draw the Sierpiński triangle (as seen in <a href="/cellular-automata#section-cellular-automata">Chapter 7</a>’s Wolfram elementary CA).</p>
   <figure>
-    <img src="images/08_fractals/08_fractals_19.png" alt=" ">
-    <figcaption> </figcaption>
+    <img src="images/08_fractals/08_fractals_19.png" alt="">
```

Comment:
@jasongao97 can you double-check this is intentional. I'm realizing that maybe I'm missing alt text for a lot of images, this is something I need to build into the process.

Reply:
> @jasongao97 can you double-check this is intentional. I'm realizing that maybe I'm missing alt text for a lot of images, this is something I need to build into the process.

Oh I think I made this change while working on replacing the images. I removed the extra space since there was no caption. Should we change it back?

Reply:
It's fine! I checked the PDF and there is no change in the build. Probably best to remove the space!